src/main/resources/static/api.yml

```diff
@@ -36,6 +36,13 @@ paths:
       tags:
         - Premises
       summary: Add a new premises
+      parameters:
+        - name: X-Service-Name
+          in: header
+          required: false
```

Comment:
Since the endpoint can never return a 2xx if this header isn't provided, feels like it should be `required: true`

Reply:
ignore me, just saw that it defaults to approved-premises if header is missing